### PR TITLE
feat: update text to be Open Staging

### PR DESCRIPTION
**Pull request by harishv7**
## Problem

Following up from @NatMaeTan 's comments from [Reviewing Build 1439 • isomerpages/isomercms-frontend](https://www.chromatic.com/test?appId=61fcc8e14c3af9003a102ac9&id=64382b0ca3055f62d610a93c)

Closes IS-225

## Solution

Update button text

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
  - Details ...
- [X] No - this PR is backwards compatible

## Test Plan

- Ensure that the buttons for all staging reads as "Open Staging"

## Screenshots

Screenshot 2023-07-12 at 3 48 44 PM
Screenshot 2023-07-12 at 3 48 29 PM
Screenshot 2023-07-12 at 3 48 22 PM

### CODE REVIEW

**Review by kishore03109 — APPROVED** [review]

**Review by seaerchin — APPROVED** [review]
Lgtm, wait for nat approval. Are we gonna work on the colour (Nat's other comments)

**Comment by harishv7** [comment]
> Lgtm, wait for nat approval. Are we gonna work on the colour (Nat's other comments)

@seaerchin the dropdown color is already fixed in the webpack PR

**Comment by NatMaeTan** [comment]
LGTM!

**Pull request merged**
